nototools/unicode_data.py

```diff
@@ -31,7 +31,11 @@
 import os
 from os import path
 import re
-import unicodedata  # Python's internal library
+import sys
+try:
+  import unicodedata2 as unicodedata  # Unicode 8 compliant c++ lib
```

Comment:
I'm not sure where @twardoch got the impression that it's written in C++...
To me it seems like plain C:
https://github.com/mikekap/unicodedata2/blob/master/unicodedata2/unicodedata.c